content/en/docs/troubleshooting.md

```diff
@@ -139,6 +139,47 @@ SHA1 Thumbprint: xxx
 
 Check the `Subject` info in the output. If it is the identity that signs the artifact, you need to add the `Subject` info into `trustedIdentities`.
 
+## When I verify an artifact, I get the error 'signature verification failed for all the signatures associated with IMAGE'
+
+This may result from one of the following issues:
+
+- Not having a trust store configured or the trust store is not configured correctly.
+- Trust store is not readable or the certificates stored in the trust store are not readable
+- no certificates were stored in trust store
+
+Rerun the verify command with `--verbose` to get more details. 
+
+The following example shows the trust store `mystore` has not be configured correctly and can't be found:
+
+```console
+$ notation verify $IMAGE -v
```

Comment:
```suggestion
$ notation verify $IMAGE --verbose
```

Reply:
I think it's also acceptable to use a simple alias of the flag as an example.

Reply:
fixed